Models/Event.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace WsuTriangle.AttendanceTracker.Models;
+
+public class Event
+{
+    public int EventId { get; set; }
+    public string EventName { get; set; } = string.Empty;
```

Comment:
An event should always have an event name. Also, try not to preface anything with "Event" if its in the event class... `event.EventName` is redundant

```suggestion
    public required  string Name { get; set; }
```